includes/class-wpcom-legacy-redirector.php

```diff
@@ -68,13 +78,22 @@ static function register_redirect_custom_capability() {
 	}
 
 	/**
-	 * Remove Bulk Edit from the Bulk Actions drop-down on the CPT's edit screen
+	 * Remove Bulk Edit from the Bulk Actions drop-down on the CPT's edit screen UI.
+	 *
+	 * @param array $actions current bulk actions available to drop-down.
+	 *
```

Comment:
WP chooses to have no linespace between `@param` and `@return` tags. See https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods as an example.

Descriptions should start with capitals, since these could be extracted into standalone sentences.